src/Makefile.win

```diff
@@ -35,9 +35,11 @@ COBJS  = $(CSRC:.c=.obj)
 
 LIBS = $(CUDA_DIR)\lib\x64\cudart_static.lib
 
+CP = copy
```

Comment:
This could be `INSTALL = copy` to indicate that we really want the `install` utility, but this platform is only guaranteed to have `copy`.